build_tools/docker/cmake-python-vulkan/Dockerfile

```diff
@@ -12,26 +12,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# An image for building IREE's tensorflow integrations using bazel and vulkan.
+# A base image for building IREE using CMake and running Vulkan tests. Requires
+# a child image to provide a Vulkan ICD.
 
-# Build using:
-# docker build --tag gcr.io/iree-oss/bazel-nvidia \
-#   build_tools/docker/bazel_nvidia/
+FROM gcr.io/iree-oss/cmake-python
 
-# Run interactively using the following, where IREE_WORKDIR is the path to your
-# local dev environment:
-# docker run -it --rm --entrypoint bash --volume ${IREE_WORKDIR}:/usr/src/iree/ \
-#   gcr.io/iree-oss/bazel-nvidia
-
-# Set up the image and working directory.
-# We start from bazel-nvidia so this image can be used to testing TensorFlow
-# integrations.
-FROM gcr.io/iree-oss/bazel-tensorflow
-
-# Additionally, we need to install the Vulkan SDK and the NVIDIA Vulkan driver.
+# It would be nice to have a separate install vulkan image and copy from that,
+# but I don't know all the files that installing the vulkan-sdk adds.
```

Comment:
Is it normal to use pronouns like I, my and mine in OSS code? Probably doesn't matter much either way.

Reply:
Not really specific to OSS, but the appropriate use of the first person is an old argument :-D I personally lean towards it where the alternative is some twisted passive voice or other acrobatics *just* to avoid the first person. I could also say "we", but I dunno maybe someone else knows how to do this (Lei?). This argument also comes up in academic papers all the time. I think avoiding the first person to seem "professional" or "formal" is a much lower priority than communicating with your reader :-P

I figure blame will usually say who the speaker is, but it usually doesn't actually matter (I wouldn't write "come ask me about this" because then someone actually has to figure out the speaker). This is a note for future devs explaining why something is the way it is and I wrote it in what felt like the most natural way. I quite frequently come across things like this where it's clearly not following what you'd expect the convention would be and I go down the rabbit hole of trying to fix it and sometimes discover the original author had a good reason for not doing so and man it would've been nice if they'd left a note. I'm happy to word it differently though :-D

/rant